docs/adr/0026-expand-subscription-platform-metadata.md

```diff
@@ -0,0 +1,81 @@
+# Expand Subscription Platform Metadata with Google Cloud Firestore
+
+- Status: proposed
+- Deciders: Ben Bangert, Barry Chen, Bianca Danforth
+- Date: 2021-06-24
+
+## Context and Problem Statement
+
+The Subscription Platform uses [Stripe Metadata][stripe-metadata] for numerous
+purposes. However, Stripe metadata has a limit on the number of values that we
+will reach in the near future. In order to continue to use metadata in the
+same fashion in the Subscription Platform, we need to overcome the limit placed
+by Stripe.
+
+[stripe-metadata]: https://stripe.com/docs/api/metadata
+
+## Decision Drivers
+
+- A higher storage limit than Stripe Metadata
+- A UI for the managers of products on the Subscription Platform to edit the
+  metadata
+- Access management for the UI
+- Updated data accessible in their respective environments without deployment
+
+## Considered Options
```

Comment:
Meant to hang this comment over here - not that folks are necessarily looking for yet another option, but how about...

[Kinto](https://docs.kinto-storage.org/en/latest/)

Pros

* Already deployed and in use at Mozilla for Remote Settings in Firefox
* Data can be updated without FxA deployments
* Multiple collections per Kinto instance - e.g. dev / stage / prod product sets
* Record creation UI and data validation via JSON schema - maybe friendlier than raw JSON
* Implements a review process for pending changes with approve / reject flow
* Maintains a changelog of updates and who was involved
* Has ACLs and groups for read / write / review access

Cons

* Backed by postgres, while FxA uses mysql
* Not as turn-key to set up as something like Firestore
* Yet another service & a new dependency for FxA

Trying to think of other cons

Reply:
Thanks, Les!  With those cons, I can imagine jbuck being in favor of Firestore.  The pros sound great though.

I'll incorporate it into the final edit. 👍